apps/cowswap-frontend/src/modules/trade/containers/TradeConfirmModal/index.tsx

```diff
@@ -123,10 +121,6 @@ function InnerComponent(props: InnerComponentProps) {
     )
   }
 
-  if (pendingTrade) {
-    return <TradeConfirmPendingContent pendingTrade={pendingTrade} onDismiss={onDismiss} />
```

Comment:
This is the only place where this component is used.

Kill `TradeConfirmPendingContent`?

Reply:
Deleted it, thanks!